### DISCUSSION TITLE
configurations pour inodes?

### DISCUSSION THREAD

**Discussion by gyom** [mila-iqia/SARC#13] [General]
On a présentement un fichier "sarc-dev.json" qui semble avoir un mandat très général, mais on a aussi "sarc-inodes.json" qui vient justement contredire le tout.

Je peux comprendre que ce n'est qu'un fichier pas complètement intégré, mais il me semble que la structure proposée dans le projet c'est de tout intégrer à "sarc-dev.json" et ajouter des sous-classes de `BaseModel` dans "config.py".

**Comment by bouthilx** [comment]
Ça devrait être mergé. En passant ça marche maintenant sur cedar aussi. Possiblement sur graham aussi.
